### PR TITLE
Added Sha1 support for Golang

### PR DESCRIPTION
**Pull request by T-Vova**

### CODE REVIEW

File: pkg/tf2pulumi/convert/tf12.go [hunk 1]
```diff
@@ -1902,6 +1902,13 @@ var tf12builtins = map[string]*model.Function{
 				ReturnType: elementType,
 			}, diagnostics
 		})),
+	"sha1": model.NewFunction(model.StaticFunctionSignature{
+		Parameters: []model.Parameter{{
+			Name: "input string",
```
Comment (lukehoban) [inline pkg/tf2pulumi/convert/tf12.go:1907]: How is this name used?  In all other cases here - it is a valid identifier, but this new one includes a space. Should this case also be a valid identifier, like `input`?

Comment (T-Vova) [reply pkg/tf2pulumi/convert/tf12.go]: Done.

**Comment by pgavlin** [comment]
This allows the PCL binding to succeed, but it doesn't add support for emitting calls to `sha1` in Go, right?

**Comment by pgavlin** [comment]
Ah, I see--looks like this PR is superseded by https://github.com/pulumi/pulumi/pull/7834

**Pull request merged**